parsl/dataflow/rundirs.py

```diff
@@ -23,7 +23,7 @@ def make_rundir(path):
         if not os.path.exists(path):
             os.makedirs(path)
 
-        prev_rundirs = glob(os.path.join(path, "[0-9]*"))
+        prev_rundirs = glob(os.path.join(path, "[0-9][0-9][0-9]"))
```

Comment:
I wrote the previous code with mistaken assumption that `[0-9]*` would match with any number of digits. Buggy as it is, it would still work in cases where you have >1000 rundirs. This code here although better, would break if there are > 1000 runs. Although not perfect, what do you think about '[0-9]*[0-9]'?

Reply:
Use `+` to mean one or more

Reply:
Ben `+` is not a wildcard supported by glob.